python/python/glide/async_commands/transaction.py

```diff
@@ -557,6 +557,62 @@ def hkeys(self: TTransaction, key: str) -> TTransaction:
         """
         return self.append_command(RequestType.Hkeys, [key])
 
+    def hrandfield(self: TTransaction, key: str) -> TTransaction:
```

Comment:
all same comments for commands, docs and data types

Reply:
I don't understand, can you please clarify what you want changed here?

Reply:
It is not relevant, since we should not update the return data type